### PR TITLE
Edit readme breadcrumbs for TOC purposes

### PR DESCRIPTION
**Pull request by KoryKessel-Mirantis**
Process of transforming the README.md file into something of a proper Table of Contents.

### CODE REVIEW

File: content/releases/README.md [hunk 1]
```diff
@@ -2,13 +2,29 @@
...
 ## MKE 4
 
+### Concepts and Architecture
+
+- Architecture
+- Configuration
+- Migration configuration
 
 ### Setup
-[Install mkectl](../docs/setup/README.md)
```
Comment (ry4nz) [inline content/releases/README.md:7]: We still need it for https://github.com/Mirantis/mke-docs/blob/main/content/docs/setup/README.md

File: content/releases/README.md [hunk 2]
```diff
@@ -2,13 +2,29 @@
 
 ## MKE 4
 
+### Concepts and Architecture
+
+- Architecture
```
Comment (ry4nz) [inline content/releases/README.md:7]: need links

File: content/releases/README.md [hunk 3]
```diff
@@ -2,13 +2,29 @@
...
+  - Installation
+  - Using the cluster
+  - k0s in AWS
+  - Uninstallation
+- [Upgrade from MKE 3](../docs/setup/upgrade-from-mke-3/README.md)
+
+### Reference
 
-[Upgrade from MKE 3](../docs/setup/upgrade-from-mke-3/README.md)
+- Authentication
```
Comment (ry4nz) [inline content/releases/README.md:23]: Would you rather to have a sperate section with all the links?  Or just add links to the following "Features" table with their deliverable releases?

File: content/docs/reference/supportbundle/support-bundle.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
+# Support bundle
```
Comment (ry4nz) [inline content/docs/reference/supportbundle/support-bundle.md:1]: This change does not belong to this PR.

File: content/releases/README.md (hunk 3 above)
Comment (KoryKessel-Mirantis) [reply content/releases/README.md:23]: Discussed in Slack.

File: content/releases/README.md (hunk 2 above)
Comment (KoryKessel-Mirantis) [reply content/releases/README.md:7]: Configured links, but for now they will not all work as not all of the pertinent content topic PRs have been reviewed and merged.
